src/test/scala/fr/polytechnique/cmap/cnam/etl/transformers/follow_up/FollowUpTransformerSuite.scala

```diff
@@ -2,14 +2,25 @@
 
 package fr.polytechnique.cmap.cnam.etl.transformers.follow_up
 
+import java.sql.Timestamp
+import scala.util.Try
+import org.mockito.Mockito.mock
 import org.apache.spark.sql.Dataset
-import org.apache.spark.sql.functions.lit
-import org.apache.spark.sql.types.TimestampType
 import fr.polytechnique.cmap.cnam.SharedContext
 import fr.polytechnique.cmap.cnam.etl.events._
 import fr.polytechnique.cmap.cnam.etl.patients.Patient
+import fr.polytechnique.cmap.cnam.etl.transformers.follow_up.FollowUpTransformerUtilities.{PatientDates, endReason, tracklossDate, tracklossDateCorrected}
 import fr.polytechnique.cmap.cnam.util.functions.makeTS
 
+case class inputDS(
```

Comment:
The naming is bad & why is this?

Reply:
Dataset in the class is naming input, this is the reason for the name

Reply:
I dont understand what you said. You can explain more please?